crux/src/Crux/FormatOut.hs

```diff
@@ -84,6 +84,9 @@ sayWhatFailedGoals skipIncompl showVars allGls =
                     else [] 
                  -- print abducts, if any
                  ++ if s /= [] then
+                      -- NB: If you update the contents of this error message,
+                      -- make sure to update the corresponding regex that
+                      -- checks for this in crux-llvm/test/Test.hs.
                       PP.pretty ("One of the following " ++ show (length s) ++ " fact(s) would entail the goal")
```

Comment:
This is a nit, but while we're in the area and since this is using `prettyprinter` to format the message, what do you think about using the latter's `plural` function instead of the text "fact(s)" here?

Reply:
That sounds fine to me. We'd need to make the regex slightly more complicated to accommodate the possibility of "fact" vs. "facts", but that is a small price to pay.